core/rawdb/accessors_chain_test.go

```diff
@@ -145,11 +145,11 @@ func TestBlockStorage(t *testing.T) {
 	} else if types.DeriveSha(types.Transactions(entry.Transactions())) != types.DeriveSha(block.Transactions()) || types.CalcUncleHash(entry.Uncles()) != types.CalcUncleHash(block.Uncles()) {
 		t.Fatalf("Retrieved body mismatch: have %v, want %v", entry, block.Body())
 	}
-	if actual, err := ReadEpochBlockNumber(db, big.NewInt(0)); err != nil {
-		t.Fatalf("Genesis epoch block number not found, error=%#v", err)
-	} else if expected := big.NewInt(0); actual.Cmp(expected) != 0 {
-		t.Fatalf("Genesis epoch block number mismatch: have %v, want %v", actual, expected)
-	}
+	//if actual, err := ReadEpochBlockNumber(db, big.NewInt(0)); err != nil {
```

Comment:
why are these tests not required?

Reply:
Because I removed the epoch number DB commit. It's originally added for resharding, but it's not actually used anywhere yet.